cart/src/components/Cart.js

```diff
@@ -0,0 +1,38 @@
+import "../styles/Cart.css";
+
+const Cart = ({ cartArray, totalPrice, handleIncDec, removeCart }) => {
+  const cards =
+    cartArray.length === 0 ? (
+      <h1 className="empty">Cart is Empty</h1>
+    ) : (
+      cartArray.map((items) => (
+        <div className="cart-items" key={items.id}>
+          <img src={items.logoURL} alt="cart" />
+          <h1>{items.heading}</h1>
+          <p>Price: {`₹${items.prices}`}</p>
```

Comment:
- I don't think both the rupee and dollar symbol are needed here.
- I also don't see the need of using string literal here, you can simply write -  `<p>Price: ₹{item.price}</p>`